pkg/driver/driver.go

```diff
@@ -16,19 +16,22 @@ package driver
 
 import (
 	"context"
+	"fmt"
 	"net"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	"github.com/dell/gocsi"
 	"github.com/onmetal/onmetal-csi-driver/cmd/options"
+	"github.com/onmetal/onmetal-csi-driver/pkg/driver/mountutils"
+	"github.com/onmetal/onmetal-csi-driver/pkg/driver/osutils"
+
 	"k8s.io/klog/v2"
-	"k8s.io/mount-utils"
-	utilexec "k8s.io/utils/exec"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 type driver struct {
-	mountUtil     *mount.SafeFormatAndMount
+	mountUtil     mountutils.MountWrapper
+	osUtil        osutils.OSWrapper
```

Comment:
Just call those two fields `mount` and `os`.

Reply:
Done